crates/cargo-test-support/src/compare.rs

```diff
@@ -108,6 +109,20 @@ pub fn assert_ui() -> snapbox::Assert {
         regex::Regex::new("home/\\.cargo/registry/src/-(?<redacted>[a-z0-9]+)").unwrap(),
     )
     .unwrap();
+    subs.insert("[TARGET]", rustc_host()).unwrap();
```

Comment:
what about this?

```suggestion
    subs.insert("[HOST]", rustc_host()).unwrap();
```

Reply:
If i see `HOST`, I think something else. `HOST_TARGET`?